lib/main.dart

```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
-    final String thumbnail;
-    final String poster;
-    final String overview;
-    final double voteAverage;
-    final List<String> genres;
-    int countLikes;
-
-    Movie({
-      required this.title,
-      required this.originalTitle,
-      required this.thumbnail,
-      required this.poster,
-      required this.overview,
-      required this.voteAverage,
-      required this.genres,
-      required this.countLikes,
-    });
-  }
+class ShowMovie extends StatelessWidget {
+  final Movie movie;
+
+  const ShowMovie(this.movie, {super.key});
 
-  class ShowMovie extends StatelessWidget {
-    final Movie movie;
-
-    const ShowMovie(this.movie, {super.key});
-
-    @override
-    Widget build(BuildContext context) {
-      return Scaffold(
-       body: Container(
-       color: Colors.black87,
-       child: Column(
-         children: <Widget> [
-            Image.asset( // Top Image
-             movie.thumbnail,
-           ),
-           Row( // Score and title
-             children: [
-               Padding(
-                 padding: const EdgeInsets.all(8.0),
-                 child: Card( // show the score
-                   shape: RoundedRectangleBorder(
-                     borderRadius: BorderRadius.circular(40), // made it circular
-                     side: const BorderSide(width: 5, color: Colors.indigo), // add a border
-                   ),
-                   color: Colors.blueAccent,
-                     child: Padding(
-                       padding: const EdgeInsets.all(15),
-                       child: Text(
-                         '${movie.voteAverage}',
-                         textScaleFactor: 2,
-                       )
-                     ),
-                 ),
-               ),
-               Expanded(
-                 child: Text( // Title
-                   movie.title,
-                   style: const TextStyle(
-                       color: Colors.white,
-                       fontWeight: FontWeight.bold,
-                       fontSize: 20.0
-                   ),
-                 ),
-               ),
-             ],
-           ),
-          SingleChildScrollView( // this permit can be scroll
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: Container(
+      color: Colors.black87,
+      child: Column(
+        children: <Widget>[
+          Image.asset(
+            // Top Image
+            movie.thumbnail,
+          ),
+          Row(
+            // Score and title
+            children: [
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Card(
+                  // show the score
+                  shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(80),
+                    // made it circular
+                    side: const BorderSide(
+                        width: 5, color: Colors.indigo), // add a border
```

Comment:
constant value , for this if the value is used just one time you cant declarate the constant in the same file outside the build , if not , you should take all those constant that you use more that once to a separated file , like lib -> utils-> ui_constants

Reply:
Also, trailing comma hehe

Reply:
i created a file call "general_constants".  i resolved the constant value and the trailing comma